src/EOS/PackedTransaction.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "Transaction.h"
+
+namespace TW::EOS {
+enum class CompressionType {
+    None = 0,
+    // ZLIB = 1,
+};
+
+NLOHMANN_JSON_SERIALIZE_ENUM( CompressionType, {
+{CompressionType::None, "none"},
+// // {CompressionType::ZLIB, "zlib"},
```

Comment:
why is this commented out?

Reply:
The serialized transaction can be compressed but not mandatory. EOS has default compression none. We kept it commented to implement in future or when the trustwallet will have zlib support. However, we will remove this for now.